WordPress/src/main/java/org/wordpress/android/models/Blog.java

```diff
@@ -497,7 +499,30 @@ public void setPlanID(long planID) {
     public String getPlanShortName() {
         return StringUtils.notNullStr(planShortName);
     }
+
     public void setPlanShortName(String name) {
         this.planShortName = StringUtils.notNullStr(name);
     }
+
+    public String getCapabilities() {
+        return capabilities;
+    }
+
+    public void setCapabilities(String capabilities) {
+        this.capabilities = capabilities;
+    }
+
+    public boolean checkCapability(Capability capability) {
+        // If a capability is missing it means the user don't have it.
+        if (capabilities == null || capabilities.isEmpty() || capability == null) {
+            return false;
+        }
+        try {
+            JSONObject jsonObject = new JSONObject(capabilities);
```

Comment:
I agree with @hypest here, and I'd go even deeper and store a custom POJO - it actually depends if these capabilities can be changed, can we add new capabilites, can we do something with these new capabilities within the app (like display them even if they won't be used by the app itself).